docs/spec_draft.md

```diff
@@ -1418,6 +1419,41 @@ function, with corner cases TBD. Numeric precision is implementation-defined.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.log_plus_one
+
+### Semantics
+
+Computes element-wise natural logarithm of `operand` tensor incremented by 1 and
```

Comment:
Proposed change rationale:
* "incremented by 1" -> "plus one" since it is formalized shortly after
* Referring to IEEE-754 specification for this compound op.
* Pointing out corner cases and numeric precision as TBD and IB respectively.
```
Computes element-wise logarithm of `operand` tensor plus one and
produces a `result` tensor. More formally,
`result[i0, ..., iR-1] = log(operand[i0, ..., iR-1] + 1)`, where `log` is the
operation from IEEE-754 specification. For complex element types, it computes a
complex log plus one, with corner cases TBD. Numeric precision is
implementation-defined.
```

Reply:
Thanks